golem_messages/cryptography.py

```diff
@@ -23,7 +23,11 @@
 elif sys.platform == 'darwin':
     # FIX PATH ON OS X ()
     # https://github.com/yann2192/pyelliptic/issues/11
-    _openssl_lib_paths = ['/usr/local/Cellar/openssl/']
+    _openssl_lib_paths = [
```

Comment:
i think its more future proof if we do a glob for this list, something like `/usr/local/Cellar/openssl/*?`

Reply:
I'd leave this as is for safety, especially since we need to migrate from `pyelliptic` anyway.